packages/server/lib/controllers/sync.controller.ts

```diff
@@ -335,6 +337,136 @@ class SyncController {
         }
     }
 
+    public async pause(req: Request, res: Response, next: NextFunction) {
+        try {
+            const { provider_config_key, connection_id } = req.query;
```

Comment:
This is a `POST`, yet we're getting the params from the query params?

Reply:
Yeah. With #972 in mind I was thinking we should have one way to specify `connectionId` and `providerConfigKey`. Otherwise they could either be in the body or query params depending on the method. Let's touch base on this quickly